src/main/java/org/cryptomator/ui/traymenu/TrayMenuBuilder.java

```diff
@@ -31,7 +31,7 @@
 public class TrayMenuBuilder {
 
 	private static final Logger LOG = LoggerFactory.getLogger(TrayMenuBuilder.class);
-	private static final String TRAY_ICON_MAC = "/img/tray_icon_mac.png";
+	private static final String TRAY_ICON_MAC = "/img/tray_icon_mac@2x.png";
 	private static final String TRAY_ICON = "/img/tray_icon.png";
```

Comment:
Please also update the tray icon for Windows and Linux, as they suffer from the same problem (see #2357). Let's hope they scale down the 2x variant correctly as well.

Reply:
We don't have `@2x` icons of `tray_icon.png`. I gave @infeo some assets recently that have a higher resolution for testing purposes, will have to see if they work out.

Reply:
It turns out there is a 32x32 png in the img folder - window_icon_32.png. Built and tested on Windows 11, seems to look ok at 100% and 200%. Pushed additional change.